extension/src/extension.ts

```diff
@@ -548,6 +556,44 @@ export class Extension extends Disposable implements IExtension {
     return this.cliCompatible
   }
 
+  private initializeGit() {
+    const cwd = getFirstWorkspaceFolder()
+    if (cwd) {
+      this.gitExecutor.init(cwd)
+      this.workspaceChanged.fire()
+    }
+  }
+
+  private async canGitInitialize(needsGitInit: boolean) {
```

Comment:
[I] Both of these need to set context values so we can show Initialize Git in a welcome view (follow-up)